hardhat.config.ts

```diff
@@ -18,6 +18,7 @@ import "./tasks/v-testnet-launch/deploy-grtPool";
 import "./tasks/v-testnet-launch/deploy-grtLiquidityWallet";
 import "./tasks/v-testnet-launch/update-grtLiquidityWallet";
 import "./tasks/v-testnet-launch/update-grtPool";
+require("hardhat-gas-reporter")
```

Comment:
@juniormp Can you replace by an `import`?

Reply:
@tcoratger sure ! Forgot about that 😆